src/index.ts

```diff
@@ -206,10 +206,14 @@ export class CdktfProviderProject extends cdk.JsiiProject {
               "sed -i 's/# CDKTF prebuilt bindings for/# CDKTF Go bindings for/' .repo/dist/go/*/README.md",
               // @see https://stackoverflow.com/a/49511949
               // eslint-disable-next-line prettier/prettier
-              `sed -i -e '/## ${isDeprecated ? 'Deprecated' : 'Available'} Packages/,/### Go/!b' -e '/### Go/!d;p; s/### Go/## Go Package/' -e 'd' .repo/dist/go/*/README.md`,
+              `sed -i -e '/## ${
+                isDeprecated ? "Deprecated" : "Available"
+              } Packages/,/### Go/!b' -e '/### Go/!d;p; s/### Go/## Go Package/' -e 'd' .repo/dist/go/*/README.md`,
               // sed -e is black magic and for whatever reason the string replace doesn't work so let's try it again:
               // eslint-disable-next-line prettier/prettier
-              `sed -i 's/### Go/## ${isDeprecated ? 'Deprecated' : 'Go'} Package/' .repo/dist/go/*/README.md`,
+              `sed -i 's/### Go/## ${
+                isDeprecated ? "Deprecated" : "Go"
+              } Package/' .repo/dist/go/*/README.md`,
```

Comment:
Nitpick but it'd be great to reverse this because IMO this actually makes it harder to read, not easier.

Reply:
Fixed it